dts/arm/ti/lm3s6965.dtsi

```diff
@@ -47,6 +47,69 @@
 			status = "disabled";
 			label = "UART_2";
 		};
+
+		gpio0: gpio@40004000 {
+			compatible = "ti,stellaris-gpio";
+			reg = <0x40004000 0xFFF>;
```

Comment:
`reg = <0x40004000 0xFFF>;`

should be:

`<0x40004000 0x1000>;`

Same comment for all other regs

Reply:
fixed